packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -6,6 +6,10 @@ $rowVerticalPadding: 11px;
 $rowHorizontalPadding: 8px;
 $compactRowVerticalPadding: 6px;
 $isPaddedMargin: 24px;
+$rowShimmerHorizontalBorder: $rowHorizontalPadding;
+$rowShimmerVerticalBorder: ($rowHeight - 7px) / 2;
```

Comment:
Should we have a var for `7px`? What does it represent?

Reply:
This represents the height of the shimmering lines to render. I assume it's logical to be moved to a var.